official/keras_application_models/benchmark_main.py

```diff
@@ -146,6 +154,18 @@ def define_keras_benchmark_flags():
       help=flags_core.help_wrap(
           "Model to be benchmarked."))
 
+  flags.DEFINE_integer(
+      name="num_images", default=_NUM_IMAGES,
+      help=flags_core.help_wrap(
+          "The number of synthetic images for training and evaluation. The "
+          "default value is 1000."))
```

Comment:
Maybe I missed something, but is this the only place this constant is used? If so then do we just want the default to be specified here?

Reply:
Yeah, we use 1000 as the default number of images for both train and eval with synthetic dataset. Will just put a value here.